sdk/servicebus/service-bus/src/util/utils.ts

```diff
@@ -513,24 +512,6 @@ export function isAbsoluteUrl(url: string) {
   return _url.startsWith("sb://") || _url.startsWith("http://") || _url.startsWith("https://");
 }
 
-/**
- * Helper method to map `DispositionStatus` to `DispositionType`
- *
- * @internal
- * @ignore
- * @param {DispositionStatus} dispositionStatus
- * @returns {(DispositionType | undefined)}
- */
-export function getDispositionType(
-  dispositionStatus: DispositionStatus
-): DispositionType | undefined {
-  if (dispositionStatus === DispositionStatus.abandoned) return DispositionType.abandon;
-  else if (dispositionStatus === DispositionStatus.completed) return DispositionType.complete;
-  else if (dispositionStatus === DispositionStatus.defered) return DispositionType.defer;
-  else if (dispositionStatus === DispositionStatus.suspended) return DispositionType.deadletter;
-  return undefined;
```

Comment:
Interesting, looks like 'renewed' as never even used. This is a nice change you're making sir.